### DISCUSSION TITLE
Command cronjob di cpanel?

### DISCUSSION THREAD

**Discussion by edwinb4li** [hotspotbilling/phpnuxbill#35] [Q&A]
halo, apakah perlu command khusus utk menjalankan cronjob phpnuxbill di cpanel?

![cara](https://user-images.githubusercontent.com/124318856/228165027-c6c3ca61-aeba-4f6f-a48e-d591b4b15b8f.jpg)

**Comment by faizakbar18** [comment]
di command isi seperti ini saja `wget -q -O- https://[nama-domain]/system/cron.php`

**Comment by masagun-unughaid** [comment]
bisa menggunakan command curl kemudian di ikuti alamat domain dan path cron php nya
